src/github.com/getlantern/flashlight/genconfig/cloud.yaml.tmpl

```diff
@@ -1,6 +1,7 @@
 # cloud.yaml contains the default configuration that's made available on the
 # internet.
 client:
+  firetweetversion: "0.0.6"
```

Comment:
OK we could do that. I feel like we'll probably start releasing a new version of FT only every few weeks though so wouldn't be such a big deal running ``genconfig`` every time